src/Compatibility/Microsoft.DotNet.ApiCompatibility/Abstractions/Mappers/AssemblySetMapper.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.CodeAnalysis;
+using System.Collections.Generic;
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
nit: license is at incorrect place

Reply:
Thanks. I have no idea how this happened 😄